biblib/views/library_view.py

```diff
@@ -391,39 +403,31 @@ def get(self, library):
                 # Now check if we can update the library database based on the
                 # returned canonical bibcodes
                 if solr.get('response'):
+                    current_app.logger.debug("sort order is set to{}".format(add_sort))
                     # Update bibcodes based on solrs response
                     updates = self.solr_update_library(
                         library_id=library.id,
                         solr_docs=solr['response']['docs']
                     )
                     if add_sort:
-                        if 'asc' in add_sort: 
-                            solr = self.timestamp_sort(solr, library.id, reverse=True)
-                        elif 'desc' in add_sort:
-                            solr = self.timestamp_sort(solr, library.id)
+                        solr = self.timestamp_sort(solr, library.id, reverse=add_sort)
 
                     documents = [i['bibcode'] for i in solr['response']['docs']]
                 else:
                     # Some problem occurred, we will just ignore it, but will
                     # definitely log it.
                     solr = SOLR_RESPONSE_MISMATCH_ERROR['body']
+                    current_app.logger.debug("sort order is set to {}".format(add_sort))
```

Comment:
`add_sort` is a boolean now - is that what you're trying to log?

Reply:
Honestly, these debug logs are not that useful anymore. Going to just remove them.